src/App.tsx

```diff
@@ -21,8 +21,16 @@ import {
 import apolloClient from './api/apolloClient';
 import { ApolloProvider } from '@apollo/client';
 import { QueryClient, QueryClientProvider } from 'react-query';
+import { ReactQueryDevtools } from 'react-query/devtools';
 
-export const queryClient = new QueryClient();
+export const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+      refetchOnMount: false
```

Comment:
This is the line that took me forever to figure out. This line stops the dashboard from refetching the data again after creating a journal entry and then going to the dashboard. I believe this is because the query for the dashboard is inactive and causing it to not trigger correctly. This is a work around for now and should be explored further. For the most up to date data, the query should be invalidated and refetched from the database. I think this is ok for now because it is not quite optimistic updating because it is done in the onSuccess of the mutation and I've included error handling.